cms/templates/settings.html

```diff
@@ -342,6 +342,15 @@ <h2 class="title-2">${_("Introducing Your Course")}</h2>
                       a_link_end=HTML("</a>")
                     )}</span>
                 </li>
+                <li class="field text" id="field-course-overview-teacher">
+                  <label for="course-overview-teacher">${_("Course teachers")}</label>
```

Comment:
Please use the same naming style `Course teachers` -> `Course Teachers`

Reply:
Done, thx!